test/Authorization.cs

```diff
@@ -0,0 +1,87 @@
+using System.Security.Claims;
+using FluentAssertions;
+using IdentityServerConfig.Components;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace IdentityServerConfig.UnitTest;
+
+[TestFixture]
+public class Authorization
```

Comment:
Maybe `AuthorizationPoliciesTests` would be a more appropriate name for this class.

Reply:
renamed in adf529e